internal/cmdget/cmdget.go

```diff
@@ -87,7 +88,12 @@ func (r *Runner) runE(c *cobra.Command, args []string) error {
 	}
 
 	if r.AutoSet {
-		if err := setters.PerformAutoSetters(r.Get.Destination); err != nil {
+		a := setters.AutoSet{
+			Writer:     c.OutOrStdout(),
+			TargetPath: r.Get.Destination,
+			ParentPath: filepath.Dir(r.Get.Destination),
```

Comment:
Can we explain this? Is destination here a reference to a file or a folder? And is ParentPath always just the parent folder?

Reply:
Commented above.